### DISCUSSION TITLE
How can I access a node app I just deployed ?

### DISCUSSION THREAD

**Discussion by Thomasevano** [coollabsio/coolify#475] [Q&A] [answered]
I have deployed a node app but I don't know what url I can call it ?

**Comment by vasani-arpit** [comment, marked as answer]
Assuming you are using version 2.x

Applications > your newly created app >  click on first icon besides the page title ![image](https://user-images.githubusercontent.com/6497827/177381912-6b3cf888-ecbe-4e32-b84d-5b42ae6dab92.png)


If you are asking how to assign the URL then you can change/set it from configuration > application > URL.

**Reply by Thomasevano** [reply]
Oh i have to first enter a custom URL with my own domain, I thought it would give me one like heroku 🤦🏻

That's why there was the message 'missing configuration' file that make sense now
